Geometry/GEMGeometryBuilder/python/gemGeometryDB_cfi.py

```diff
@@ -10,3 +10,7 @@
     alignmentsLabel = cms.string(''),
     applyAlignment = cms.bool(False)
 )
+
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+
+run3_GEM.toModify(GEMGeometryESModule, applyAlignment = True)
```

Comment:
Done we also need
```
phase2_GEM.toModify(GEMGeometryESModule, applyAlignment = True)
```
as well?

Reply:
After private discussion offline between PR author and @cms-sw/alca-l2 it was decided to postpone the update of phase2 to a later moment.